src/windows/BarcodeScannerProxy.js

```diff
@@ -440,7 +441,20 @@ module.exports = {
         function startPreview() {
             return findCamera()
             .then(function (id) {
-                var captureSettings = new Windows.Media.Capture.MediaCaptureInitializationSettings();
+                var captureSettings;
+
+                try {
+                    captureSettings = new Windows.Media.Capture.MediaCaptureInitializationSettings();
+                } catch (e) {
+                    console.error(e);
```

Comment:
I'd remove the logging here - the error object will be available in error handler anyway.